internal/provider-integration/command_list.go

```diff
@@ -0,0 +1,72 @@
+package providerintegration
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *command) newListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List provider integrations.",
+		Long:  "List Provider Integrations, optionally filtered by cloud provider.",
+		Args:  cobra.NoArgs,
+		RunE:  c.list,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "List Provider Integrations in current environment.",
+				Code: "confluent provider-integration list",
+			},
+			examples.Example{
+				Text: "List Provider Integrations in environment env-abcdef.",
+				Code: "confluent provider-integration list --environment env-abcdef",
+			},
+			examples.Example{
+				Text: "List Provider Integrations in current environment with AWS as cloud provider.",
+				Code: "confluent provider-integration list --cloud aws",
+			},
```

Comment:
We should leave this out for now since it doesn't do anything (only AWS is supported).
```suggestion
```